frontend/encore/simple-example.rst

```diff
@@ -59,27 +65,21 @@ To build the assets, run the following if you use the Yarn package manager:
 
 .. code-block:: terminal
 
-    # compile assets once
-    $ yarn encore dev
+    # compile assets and automatically re-compile when files change
+    $ yarn watch
 
-    # or, recompile assets automatically when files change
-    $ yarn encore dev --watch
-
-    # on deploy, create a production build
-    $ yarn encore production
-
-If you use the npm package manager, run the following commands instead:
```

Comment:
We recently updated all front-end docs to always display both `yarn` and `npm` commands, because there's no clear winner between both, so we get lots of complains when we only display the commands of one of them.

So, could we re-add the `npm` commands here? Thanks!

Reply:
I added back a little bit, but not this entire section. Let me know what you think. When I was reading through this, I thought it looked a little sloppy to have an entire `yarn` section and THEN an entire `npm` section with the same commands. But, you've experienced more of the history of trying to get this issue correct, so let me know if you think my changes don't go far enough :)

Reply:
OK, I understand. Let's keep it as you said. Thanks for the tweaks.